docs/design/3_1_2_platform_management.tex

```diff
@@ -11,14 +11,16 @@ \subsubsection{Subsystem Overview}
 \end{figure}
 
 \subsubsection{Device Selection}
-The selected device to implement the platform management system is the Raspberry Pi 4. As seen in Figure \ref{rpi}, the Raspberry Pi 4 is a small single-board computer which hosts an ARM Cortex A72 CPU, a CSI camera interface, 40 serial pins, WiFi/Bluetooth, and a gigabit Ethernet port. The Raspberry Pi 4 was selected over other single-computer platforms (such as Arduino/Beagleboard) as it:
+The selected device to implement the platform management system is the Raspberry Pi 4. As seen in Figure \ref{rpi}, the Raspberry Pi 4 is a small single-board computer which hosts a quad-core ARM Cortex A72 CPU, a CSI camera interface, 40 serial pins, WiFi/Bluetooth, and a gigabit Ethernet port. The Raspberry Pi 4 was selected over other single-computer platforms (including Arduino/Beagleboard/Orange Pi) as it:
```

Comment:
Active voice:

```suggestion
The selected device to implement the platform management system is the Raspberry Pi 4. As seen in Figure \ref{rpi}, the Raspberry Pi 4 is a small single-board computer that hosts a quad-core ARM Cortex A72 CPU, a CSI camera interface, 40 serial pins, Wi-Fi/Bluetooth, and a gigabit Ethernet port. We choose the Raspberry Pi 4 over other single-computer platforms (including Arduino/Beagleboard/Orange Pi) because the Raspberry Pi 4:
```